### ISSUE TITLE
Is it possible to use custom SVG tooltip inside VictoryTooltip (wrap flyoutComponent to an existing VictoryTooltip)?

### ISSUE THREAD

**Issue by alexfrize** [FormidableLabs/victory#1902]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

Describe the issue you're seeing, and what you expect the behavior to be.

### Reproduction

Here is the example (line 224)
https://codesandbox.io/s/objective-night-2helz?file=/index.js
I know it is a very (very) bad code (draft). Apologize for it. But I need to find the visual solution first, then refactor it.

### Description
Is there any way to change the text to an SVG container or HTML?
In other words, I would like to use the existing VictoryTooltip and wrap CustomTooltip in it. But currently, CustomTooltip is replacing existing VictoryTooltip styles (lines 224-232).
Something like html="" or svg="" instead of text=""?

**Comment by boygirl** [comment]
@alexfrize thanks for this issue. The tooltips you have working look good, even if the `text=""` work around is slightly ugly. I'm not 100% sure I understand what you're trying to do, but I think the confusion might be around the fact that `VictoryTooltip` is set up to render both a `flyoutComponent` and a `labelComponent`. For your uses, they make sense as a single custom component, but that's why you're getting two separate things when you don't no-op the `text` prop.

**Comment by beccanelson** [comment]
I'm going to close this issue for now, but we can re-open if there are ongoing issues here.
